parsl/executors/high_throughput/executor.py

```diff
@@ -346,9 +346,9 @@ def validate_resource_spec(self, resource_specification: dict):
         if resource_specification:
             raise InvalidResourceSpecification(
                 set(resource_specification.keys()),
-                ("HTEX does not support the supplied resource_specifications."
+                ("HTEX does not support the supplied resource_specifications. "
                  "For MPI applications consider using the MPIExecutor. "
-                 "For specifications for core count/memory/walltime, consider using WorkQueueExecutor. ")
+                 "For specifications for core count/memory/walltime, consider using WorkQueueExecutor.")
```

Comment:
FWIW, I have found that putting the space(s) to join lines at the _beginning_ of the following line easier to see and not leave this kind of detritus.  Obviously not for this PR, but as a habit/mental trick to myself.  :shrug: